test/lib-test/jdk/test/lib/format/ArrayDiffTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2021, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should it just modify the second year, here is `2021`, to `2023`?

Reply:
Yes, good catch!

Reply:
The line must only two years. Like so:

` * Copyright (c) 2020, 2023, Oracle and/or its affiliates. All rights reserved.`